src/applications/static-pages/authenticated-homepage/index.js

```diff
@@ -0,0 +1,20 @@
+// Node modules.
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+
+export default (store, widgetType) => {
+  const root = document.querySelector(`[data-widget-type="${widgetType}"]`);
+  if (root) {
+    import(/* webpackChunkName: "authenticated-homepage" */
+    './components/App/App').then(module => {
+      const App = module.default;
+      ReactDOM.render(
+        <Provider store={store}>
+          <App />
+        </Provider>,
+        root,
+      );
+    });
+  }
+};
```

Comment:
Since this is being lazy loaded instead of the application bundle, it's going to be (weight of static-pages.js + weight of auth-homepage.js) before the page is rendered. Most of the static-pages bundle is wasteful. Instead of a widget, I would consider creating your own app entry point like suggested in https://github.com/department-of-veterans-affairs/vets-website/pull/13987.